xarray/core/combine.py

```diff
@@ -371,7 +371,7 @@ def _nested_combine(
 
 # Define type for arbitrarily-nested list of lists recursively
 # Currently mypy cannot handle this but other linters can (https://stackoverflow.com/a/53845083/3154101)
```

Comment:
This seems to work now judging from the removed type ignore.
Good to know:)

Reply:
Removed the outdated comment.